manifest.toml

```diff
@@ -65,14 +65,14 @@ ram.runtime = "50M"
     [resources.permissions]
     main.url = "/"
     main.show_tile = false
-    server_api.url = "__DOMAIN__/_matrix "
```

Comment:
@ericgaspar this one was pretty tricky to find out, but these spaces were breaking the path detection so all requests were going through the SSO (and triggering CORS issues).